src/Linux/Avalonia.LinuxFramebuffer/Output/FbdevOutput.cs

```diff
@@ -90,17 +126,17 @@ public PixelSize PixelSize
             get
             {
                 fb_var_screeninfo nfo;
-                if (-1 == NativeUnsafeMethods.ioctl(_fd, FbIoCtl.FBIOGET_VSCREENINFO, &nfo))
```

Comment:
The general rule is to not submit formatting-only changes in PRs

Reply:
No problem, will undo